src/components/Search.js

```diff
@@ -10,12 +11,18 @@ const allSuggestions = structure.sections
   .reduce((acc, val) => acc.concat(val), [])
   .sort();
 
-export default () => {
+const SearchButton = ({ setSearchOpen, searchOpen }) => {
```

Comment:
I think from this components perspective, these could just be `open` and `setOpen`.

Reply:
I agree. Let me change that.